executor/distsql.go

```diff
@@ -512,9 +512,73 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, kvRanges []k
 		if err != nil {
 			e.feedback.Invalidate()
 		}
+<<<<<<< HEAD
```

Comment:
Please fix the merge conflict

Reply:
I found the code structure is completely different between release-5.0 with other branches. And it's hard to fix in release-5.0 with same way. Maybe we have two ways to do this:

1. The dynamic prune isn't GA in release-5.0, just close this cherry-pick.
2. fix it by change the plan, don't push Limit into index look up.

Which one are you prefer? @mjonss